app/presenters/relationship_label.rb

```diff
@@ -14,7 +14,8 @@ def initialize(relationship, is_reverse = false)
 
   def display_date_range
     if start_date.nil? && end_date.nil?
-      return '(past)' if is_current == false
+      return '(past)' unless is_current
```

Comment:
This should actually remain ` is_current == false`

It's another quirk of our data model. Although is_current is a boolean column, we differentiate between `nil` and `false`. `false` means the relationship is in the past, but `nil` indicates that if the state is _unknown_ and we shouldn't display the _past_ label.

Reply:
Right, thanks for the explanation. I think this should be encapsulated in some way so that it's not necessary to know the unusual logic here. E.g. on relationship:

```
def temporal_status
  case is_current
  when true
    :current
  when false
    :past 
  when nil
    :unknown
  end
end
```